Gems/Atom/RHI/Code/Source/RHI/RHIUtils.cpp

```diff
@@ -446,7 +446,7 @@ namespace AZ
 #endif
             char gpuMemDumpFileName[128];
             strftime(gpuMemDumpFileName, sizeof(gpuMemDumpFileName), "%Y-%m-%d.%H-%M-%S", &today);
-            AZStd::string filename = AZStd::string::format("%s/GpuMemoryLog_%s.%" PRIu64 ".json", path.c_str(), gpuMemDumpFileName, ltime);
+            AZStd::string filename = AZStd::string::format("%s/GpuMemoryLog_%s.%ld.json", path.c_str(), gpuMemDumpFileName, ltime);
```

Comment:
This is not going to work on platforms where time_t is a `long long` such as Linux.
https://stackoverflow.com/questions/2792551/what-primitive-data-type-is-time-t
The ltime variable should also be cast to a `long`
```suggestion
            AZStd::string filename = AZStd::string::format("%s/GpuMemoryLog_%s.%ld.json", path.c_str(), gpuMemDumpFileName, static_cast<long>(ltime));
```